css-shapes-1/Overview.bs

```diff
@@ -423,106 +457,64 @@ Supported Shapes</h3>
 			define rounded corners for the rectangle
 			using the 'border-radius' shorthand syntax.)
 
-		<dt><dfn>circle()</dfn> =
-			circle( <<shape-radius>>? [ at <<position>> ]? )
-		</dt>
+		<dt><dfn>circle()</dfn>
 		<dd>
-			<ul>
-				<li>
-					The shape-radius argument represents
-					<strong>r</strong>, the radius
-					of the circle.
-					Negative values are invalid.
-					A percentage value here
-					is resolved from the used width and height
-					of the <a>reference box</a> as <br>
-					<code>sqrt(<em>width</em><sup>2</sup>+<em>height</em><sup>2</sup>)/sqrt(2)</code>.
-				</li>
-				<li>
-					The position argument defines
-					the center of the circle.
-					This defaults to center if omitted.
-				</li>
-			</ul>
-		</dd>
-		<dt><dfn>ellipse()</dfn> =
-			ellipse( [ <<shape-radius>>{2} ]? [ at <<position>> ]? )
-		</dt>
+			* The <<radial-size>> argument defines
+				the circle's radius.
```

Comment:
Was there a reason for omitting “Negative values are invalid” here and for ellipse()?

Reply:
Already defined as part of the production, which has bounds on the `<length-percentage>`. It didn't need to be reproduced.